src/test/java/dev/vasyl/proj/config/CustomMySqlContainer.java

```diff
@@ -0,0 +1,28 @@
+package dev.vasyl.proj.config;
+
+import org.testcontainers.containers.MySQLContainer;
+
+public class CustomMySqlContainer extends MySQLContainer<CustomMySqlContainer> {
+    private static final String DB_IMAGE = "mysql:8";
+    private static CustomMySqlContainer mySqlContainer;
+    public CustomMySqlContainer() {
```

Comment:
Do you have checkstyle?
```suggestion
    private static CustomMySqlContainer mySqlContainer;
    
    public CustomMySqlContainer() {
```